### ISSUE TITLE
Create model tests

### ISSUE THREAD

**Issue by Andre-Pham** [FIT4002-TEAM9/Leaf#119]
The group presentation is a big deal (20%), and we need model tests.

You'll need to do research on how to implement this. Then you'll need to implement them - create a branch called logic-tests. They don't need to be perfect, as long as they're enough to talk about during the presentation.

This involves testing all of our functionality in our classes (see our model directory in our project). Stuff like our patient class, employee classes, etc.

Basically you're calling methods on our objects and making sure that the logic worked. For instance, create a patient and a worker object, allocate the patient to the worker, (using the method on the worker), and see if they were added. Or creating a new changelog instance, logging a new change, and checking to see it was added. Stuff like that.
